tests/integration/incremental_indexing/test_incremental_indexing.py

```diff
@@ -1,91 +1,60 @@
 import os
-import numpy as np
+
 from jina.flow import Flow
-from jina.proto import jina_pb2
-from jina.drivers.helper import array2pb
 from jina.executors import BaseExecutor
 from jina.executors.indexers.vector import NumpyIndexer
 from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from tests.integration.incremental_indexing import random_workspace, get_duplicate_docs
 
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 
 
-def get_duplicate_docs(num_docs=10):
-    result = []
-    unique_set = set()
-    for idx in range(num_docs):
-        doc = jina_pb2.Document()
-        content = int(idx / 2)
-        doc.embedding.CopyFrom(array2pb(np.array([content])))
-        doc.text = f'I am doc{content}'
-        result.append(doc)
-        unique_set.add(content)
-    return result, len(unique_set)
-
-
-def test_incremental_indexing_vecindexers(tmpdir):
-    os.environ['JINA_TEST_INCREMENTAL_INDEX_WORKSPACE'] = str(tmpdir)
-    total_docs = 10
+def test_incremental_indexing_sequential_indexers(random_workspace):
```

Comment:
you can have `duplicated_docs` as well as a `fixture` (learning from @bwanglzu), it helps at showing what is being tested and what is input from the outside

Reply:
Good point! The fixture is really handy.